app/controllers/callbacks_controller.rb

```diff
@@ -38,23 +31,20 @@ def receive_discord_callback
       Rails.logger.info("User: #{l.to_json}")
     end
 
-    # TODO: DRY this up
     users.yield_self do |oldest_user, *newer_users|
```

Comment:
this whole loop feels a little convoluted but not exactly sure what it's doing so hard to pin down why... maybe needs a comment explaining?